src/bin/common.rs

```diff
@@ -604,6 +604,22 @@ fn parse_config(matches: &ArgMatches<'_>) -> Result<EncoderConfig, CliError> {
         f64
       )
       .expect("Cannot parse the mastering display option");
+
```

Comment:
Maybe add a reference to spec,
I think it is "6.7.4. Metadata high dynamic range mastering display color volume semantics"

Where it defines the details for chromaticity X and Y

Reply:
I added a reference in a comment and in the warning but not sure if you want it both places.